CliWrap/Cli.cs

```diff
@@ -58,6 +59,18 @@ private Process CreateProcess(string arguments)
                 },
                 EnableRaisingEvents = true
             };
+
+            if (env != null)
+            {
+                foreach (var item in env)
+                {
+#if NET45
```

Comment:
This works in .net standard 2.0 too so use `#if NET45 && NETSTANDARD2_0` or however you write the net std moniker.

Reply:
I have absolutely no idea how those .NET things work, so sorry about that.

Reply:
Don't worry about it

Reply:
When compiling .NET Standard it says that the type isn't found, I guess it doesn't work.

Reply:
It definitely works for me. You sure you have the net standard 2.0 sdk?

Reply:
I don't think so, I'll install it and try again.